src/Tasks/vmOperations/task.json

```diff
@@ -142,6 +142,15 @@
             "helpMarkDown": "Name of the snapshot of the virtual machines."
         },
         {
+            "name": "snapshotMemory",
+            "type": "boolean",
+            "label": "Snapshot Memory",
+            "defaultValue": "true",
+            "required": false,

```

Comment:
Instead of changing tl.getInput you can change required attribute in task.json to true. Once this is done I will merge the PR